tools/autograd/derivatives.yaml

```diff
@@ -959,9 +959,11 @@
 
 - name: mvlgamma(Tensor self, int p) -> Tensor
   self: mvlgamma_backward(grad, self, p)
+  result: auto_element_wise
```

Comment:
Btw, here's a list of all the ops that appear in this PR: https://docs.google.com/spreadsheets/d/16DgjrIyzLbNMOIRNF_nV7qxTFqBFapRwisoNftip2s0/edit?usp=sharing